sympy/series/tests/test_series.py

```diff
@@ -328,12 +328,12 @@ def test_issue_20697():
     Q = (p_0 + (p_1 + (p_2 + p_3/y)/y)/y)/(1 + ((p_3/(b_0*y) + (b_0*p_2\
         - b_1*p_3)/b_0**2)/y + (b_0**2*p_1 - b_0*b_1*p_2 - p_3*(b_0*b_2\
         - b_1**2))/b_0**3)/y)
-    assert Q.series(y, n=3) == b_2*y**2 + b_1*y + b_0 + O(y**3)
+    assert Q.series(y, n=3).ratsimp() == b_2*y**2 + b_1*y + b_0 + O(y**3)
 
 
 def test_issue_21245():
     fi = (1 + sqrt(5))/2
-    assert (1/(1 - x - x**2)).series(x, 1/fi, 1) == \
-        (36/(-36*sqrt(5) - 80) + 16*sqrt(5)/(-36*sqrt(5) - 80))/(x - 1/(S.Half\
-        + sqrt(5)/2)) - 1220*sqrt(5)/(-6100*sqrt(5) - 13640) - 2728\
-        /(-6100*sqrt(5) - 13640) + O(x - 2/(1 + sqrt(5)), (x, 2/(1 + sqrt(5))))
+    assert (1/(1 - x - x**2)).series(x, 1/fi, 1).factor() == \
```

Comment:
The result could probably be improved. I tried with `radsimp` but that raises
```
ValueError: expecting a Symbol but got _Dummy_628 - sqrt(5)/2 + 1/2 - (1/2 - sqrt(5)/2)
```
It looks like this would come from this line: https://github.com/sympy/sympy/blob/1a9577f9220fc24372ad4bd6d97059759a6b665c/sympy/series/order.py#L193
because
```
(Pdb) p rs
{_Dummy_628 - sqrt(5)/2 + 1/2 - (1/2 - sqrt(5)/2): x - sqrt(5)/2 + 1/2}
```
Obviously, some simplification is necessary for `v - point[0]`.